sn_cli/Cargo.toml

```diff
@@ -14,6 +14,9 @@ version = "0.77.0"
 path="src/main.rs"
 name="safe"
 
+[features]
+metrics = ["sn_logging/process-metrics"]
```

Comment:
just a remind that this feature need to be enabled for the public testnet run

Reply:
Yes! I will ping Chris about including the feature for the tesnet release. (if we have automated the release process I mean)

Reply:
@RolandSherwin if this is included in the next testnet and you want me to get `vdash` ready for it, can you give me an example of the logfile trace output?

Reply:
Sure @happybeing! This is the JSON object that is currently being emitted,
```json
{"physical_cpu_threads":12,"system_cpu_usage_percent":8.87408,"system_total_memory_mb":33517.777,"system_memory_used_mb":14960.186,"system_memory_usage_percent":44.633587,"network":{"interface_name":"enp0s31f6","bytes_received":1350,"bytes_transmitted":53303,"total_mb_received":2376.993,"total_mb_transmitted":749.26776},"process":{"cpu_usage_percent":4.6299543,"memory_used_mb":38.64576,"bytes_read":0,"bytes_written":16384,"total_mb_read":0.0,"total_mb_written":4.251648}}
```
Just a note that the internal `network, process` objects might be empty if there were some errors while obtaining the corresponding `network` or `process` metrics.


Also, I just looked into `vdash` and it seems really cool! Will definitely try it out later :smile:

Reply:
@RolandSherwin  Can you cut and paste the output from a logfile as I'm not sure the format ends up as JSON when printed? Thanks.

Reply:
Yeah sure, this is the entire log line.
```
[2023-06-05T12:29:59.680321Z TRACE sn_logging::metrics] {"physical_cpu_threads":12,"system_cpu_usage_percent":11.24604,"system_total_memory_mb":33517.777,"system_memory_used_mb":16400.195,"system_memory_usage_percent":48.929844,"network":{"interface_name":"enp0s31f6","bytes_received":1774,"bytes_transmitted":37947,"total_mb_received":2367.518,"total_mb_transmitted":717.744},"process":{"cpu_usage_percent":1.2671595,"memory_used_mb":35.004417,"bytes_read":0,"bytes_written":8192,"total_mb_read":0.0,"total_mb_written":0.1024}}
```